components/account/home/AccountDetails.vue

```diff
@@ -22,13 +34,41 @@ const { t } = useI18n();
       <div class="space-y-5">
         <RuiTextField
           id="email"
-          v-model="email"
+          v-model="username"
           disabled
+          class="[&_input]:!text-rui-text"
           variant="outlined"
-          :label="t('auth.common.email')"
+          :label="t('auth.common.username')"
           hide-details
           color="primary"
+          append-icon="lock-line"
         />
+        <div>
+          <RuiTextField
+            id="email"
+            v-model="email"
+            disabled
+            class="[&_input]:!text-rui-text"
+            variant="outlined"
+            :label="t('auth.common.email')"
+            hide-details
+            color="primary"
+            append-icon="lock-line"
+          />
+          <div class="text-rui-text-secondary pt-2 px-3 text-xs flex">
+            To change your email, please contact our support tem via
```

Comment:
shouldn't this be extracted tot the resource file? It also has a small typo (`tem`)